### PR TITLE
Net

### PR DESCRIPTION
**Pull request by Olexandr-Sh**
Created net for customizing position of elements in our project.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,29 +1,161 @@
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+img {
```
Comment (Plakhotenko) [inline css/main.css]: Please, add video to this css rule.

File: css/main.css [hunk 2]
```diff
@@ -1,29 +1,161 @@
...
-    rgba(213, 32, 71, 0.8) 0%,
-    rgba(238, 61, 27, 0.8) 100%
-  );
+      90deg,
+      rgba(213, 32, 71, 0.8) 0%,
+      rgba(238, 61, 27, 0.8) 100%
+    ),
+    url(/images/Rectangle_3.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
```
Comment (Plakhotenko) [inline css/main.css]: Don't we need `background-size: cover`?

Comment (Plakhotenko) [inline css/main.css]:
```suggestion
  background-size: cover;
```

File: css/main.css [hunk 3]
```diff
@@ -39,13 +171,13 @@ body {
   position: relative;
   z-index: 1;
 }
-.login_button:hover {
+.login_button_sml:hover {
```
Comment (Plakhotenko) [inline css/main.css:174]: Maybe login_button_sm?

File: css/main.css [hunk 4]
```diff
@@ -111,14 +246,28 @@ body {
...
+.block_for_section_1 {
+  margin-top: 140px;
+  margin-bottom: 102px;
+}
 .text_section_1 {
+  width: 445px;
+  height: 115px;
   font-size: 16px;
-  line-height: 22px;
+  line-height: 140%;
```
Comment (Plakhotenko) [inline css/main.css]:
```suggestion
  line-height: 1.4;
```

File: css/main.css [hunk 5]
```diff
@@ -111,14 +246,28 @@ body {
...
 }
 .video {
   width: 540px;
   height: 321px;
   border-radius: 16px;
+  margin-top: 94px;
+  margin-bottom: 94px;
```
Comment (Plakhotenko) [inline css/main.css]: It should be paddings for section, not margins for video.

File: css/main.css [hunk 6]
```diff
@@ -225,23 +467,52 @@ body {
...
+.mail_list_item {
+  margin-bottom: 8px;
+  padding-top: 2px;
+  padding-bottom: 2px;
+}
+.tel_list_item {
+  margin-bottom: 20px;
 }
 .li_serv {
   color: #000000;
```
Comment (Plakhotenko) [inline css/main.css]: ?

**Comment by Plakhotenko** [comment]
Good work!

File: css/main.css (hunk 1 above)
Comment (Olexandr-Sh) [reply css/main.css]: I added this css rule to video.

File: css/main.css (hunk 2 above)
Comment (Olexandr-Sh) [reply css/main.css]: You are right, it would be better)

Comment (Olexandr-Sh) [reply css/main.css]: I agree with you, after "bs: cover" it will be redundant.

File: css/main.css (hunk 3 above)
Comment (Olexandr-Sh) [reply css/main.css:174]: Hmm.. I changed it earlier, maybe then I had forgotten to push my version into Github.

File: css/main.css (hunk 4 above)
Comment (Olexandr-Sh) [reply css/main.css]: Ok, I understand what do you mean.

File: css/main.css (hunk 5 above)
Comment (Olexandr-Sh) [reply css/main.css]: I've deleted these margins and added a new class with "paddings" to div element which covered my video.
.video-wrapper {
  padding-top: 94px;
  padding-bottom: 94px;
}

File: css/main.css (hunk 6 above)
Comment (Olexandr-Sh) [reply css/main.css]: What do you mean?

**Review by Plakhotenko — APPROVED** [review]

**Pull request merged**